portal/src/graphql/portal/CreateOAuthClientScreen.tsx

```diff
@@ -48,16 +48,8 @@ function constructFormState(
   return {
     clients: config.oauth?.clients ?? [],
     newClient: {
-      name: undefined,
       x_application_type: "spa",
       client_id: genRandomHexadecimalString(),
-      redirect_uris: [],
-      grant_types: ["authorization_code", "refresh_token"],
-      response_types: ["code", "none"],
-      access_token_lifetime_seconds: undefined,
-      refresh_token_lifetime_seconds: undefined,
-      post_logout_redirect_uris: undefined,
-      issue_jwt_access_token: true,
```

Comment:
issue_jwt_access_token should always be true, regardless of `x_application_type`